components/mpas-ocean/src/shared/mpas_ocn_wetting_drying.F

```diff
@@ -355,21 +362,38 @@ subroutine ocn_wetting_drying_wettingVelocity(layerThickEdgeFlux, layerThickness
       integer :: cell1, cell2, iEdge, iCell, k, i
 
       real (kind=RKIND) :: divOutFlux
-      real (kind=RKIND) :: layerThickness
+      real (kind=RKIND) :: columnThickness
+      real (kind=RKIND) :: hCrit, hRampMin, hRampMax, hEdgeTotal
 
       character (len=100) :: log_string
 
+      integer:: cellDummy(2), cellCur, CellNei, sshCur, sshNei
```

Comment:
@sbrus89 One more thing: `sshCur` and `sshNei` should be real, right?

Reply:
Yes, that's correct. Thanks for catching that.